lib/IRGen/GenBuiltin.cpp

```diff
@@ -1345,5 +1345,15 @@ void irgen::emitBuiltinCall(IRGenFunction &IGF, const BuiltinInfo &Builtin,
     return;
   }
 
+  if (Builtin.ID == BuiltinValueKind::PackCount) {
```

Comment:
Can we lower away the builtin at the SIL level and turn it into a SIL `pack_count` instruction?